Functions.Templates/Templates/SqlInputBinding-Python/function.json

```diff
@@ -0,0 +1,26 @@
+{
+    "scriptFile": "__init__.py",
+    "bindings": [
+      {
+        "type": "httpTrigger",
+        "direction": "in",
+        "name": "req",
+        "authLevel": "anonymous",
+        "route": "getproducts/{cost}"
+      },
+      {
+        "direction": "in",
+        "type": "sql",
+        "name": "rowList",
+        "commandText": "SELECT * FROM Products WHERE Cost = @Cost",
```

Comment:
`SELECT * FORM [dbo].[table1]`

Reply:
I think this should be `SELECT * FROM object` so that it gets filled in by the template too.

Reply:
Do we know how the template substitutions work for these worker languages? Because the C# one is a lot more complex - so for example do we know that it's able to do a partial replace like that? The JS/TS samples we have right now just assume the user has to enter the entire string but that's not really great since that relies on the user knowing what this value is going to be used for.